src/handlers/pinata.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  // @ts-expect-error ts-migrate(2305) FIXME: Module '"react-native-dotenv"' has no exported mem... Remove this comment to see the full error message
+  PINATA_API_KEY,
+  // @ts-expect-error ts-migrate(2305) FIXME: Module '"react-native-dotenv"' has no exported mem... Remove this comment to see the full error message
+  PINATA_API_SECRET,
+  // @ts-expect-error ts-migrate(2305) FIXME: Module '"react-native-dotenv"' has no exported mem... Remove this comment to see the full error message
+  PINATA_API_URL,
+  // @ts-expect-error ts-migrate(2305) FIXME: Module '"react-native-dotenv"' has no exported mem... Remove this comment to see the full error message
+  PINATA_GATEWAY_URL,
+} from 'react-native-dotenv';
+import RNFS from 'react-native-fs';
+
+type UploadFilesResponse = {
+  IpfsHash: string;
+  PinSize: number;
+  Timestamp: string;
+  isDuplicate: boolean;
+};
+export type UploadImageReturnData = UploadFilesResponse & {
+  url: string;
+};
+
+export async function uploadImage({
+  path,
+  filename,
+  mime,
+}: {
+  filename: string;
+  path: string;
+  mime: string;
+}): Promise<UploadImageReturnData> {
+  const response = await RNFS.uploadFiles({
+    files: [
+      {
+        filename,
+        filepath: path,
+        filetype: mime,
+        name: 'file',
+      },
+    ],
+    headers: {
+      pinata_api_key: PINATA_API_KEY,
+      pinata_secret_api_key: PINATA_API_SECRET,
```

Comment:
should we create a rainbow api key / secret?

Reply:
wym? this api key & secret is coming from rainbow's pinata

Reply:
hm that I don't see it on our dev keys, we need to add it there too probably from a rainbow team account

Reply:
if you did already we need to push it to our dev keys